react/index.js

```diff
@@ -6,7 +6,9 @@ module.exports = {
     'react@^15.4.1',
     'react-dom@^15.4.1'
   ],
-  devDependencies: [],
+  devDependencies: [
+    'react-hot-loader@^3.0.0-beta.6',
```

Comment:
This has to be a production dependency right?  It then chooses whether to be a noop or actually do things based on `process.env.NODE_ENV === 'production'`